src/test/utils/numa_memory_resource_test.cpp

```diff
@@ -39,4 +39,20 @@ TEST_F(NUMAMemoryResourceTest, BasicAllocate) {
   EXPECT_EQ(get_node_id_of(vec.data()), numa_node);
 }
 
+TEST_F(NUMAMemoryResourceTest, AreEqual) {
+#if HYRISE_NUMA_SUPPORT
+  const int numa_node = numa_max_node();
+#else
+  const int numa_node = 1;
+#endif
+
+  // Two memory_resources compare equal if and only if memory allocated from one memory_resource can be deallocated fro
+  // the other and vice versa.
+
+  auto resource_a = NUMAMemoryResource(numa_node, "foo");
```

Comment:
the name is irrelevant, right? any two NUMAMemoryResources will be non-equal?

Reply:
In the current implementation, yes. The name is just something that the underlying pgasus lib wants.